widgetastic_manageiq/__init__.py

```diff
@@ -2322,3 +2327,58 @@ def fill(self, values):
 
     def read(self):
         return self.all_shortcuts
+
+
+class DynamicTable(VanillaTable):
```

Comment:
In this case the table doesn't support the other methods provided by manageiq.Table - there are no check_all or row checkboxes, nor are the editable tables sortable.

For these reasons I'd like to keep them separate, though I do understand your want to minimize the class sprawl. If you feel strongly about minimizing the classes I could be convinced to combine them.